### ISSUE TITLE
newruntime + threadpool: undeclared "deepCopy"

### ISSUE THREAD

**Issue by mratsim** [nim-lang/Nim#11882]
Threadpool cannot be imported with newruntime, deepCopy is not declared.

```Nim
import threadpool

echo "Hello world!"
# compile with "--newruntime --threads:on"
```

```
.../Nim/lib/pure/concurrency/threadpool.nim(254, 12) Error: undeclared identifier: 'deepCopy'
```

**Comment by GordonBGood** [comment]
I think you may be able to close this as recent commits to do with making the `newruntime` switch work with "--threads:on" seem to have fixed it, at least when I run it on the 19-August-2019 nightly DEVEL build on Windows 64-bit.

**Comment by mratsim** [comment]
Indeed, it was fixed by bebda3851f0f9d3433c4df773bd6984a009a045f which had the wrong name (fix #1188 instead of #11882)
